octoprint_mrbeam/iobeam/temperature_manager.py

```diff
@@ -70,14 +70,16 @@ def _on_mrbeam_plugin_initialized(self, event, payload):
         self._iobeam = self._plugin.iobeam
         self._analytics_handler = self._plugin.analytics_handler
         self._one_button_handler = self._plugin.onebutton_handler
-
+        self._subscribe()
```

Comment:
This change is not related to the fix
it is done because we shouldn't start the timer before all inits are done and the subscribe is and important step during init